### PR TITLE
feat(sankey-viz): allow sort by metric and bump superset-ui packages

### PR DESCRIPTION
**Pull request by ktmud**
### SUMMARY

Allow sankey chart to sort by metric and bump superset-ui packages to bring in multiple changes by @etr2460 and @villebro :

Diff: https://github.com/apache-superset/superset-ui/compare/v0.15.10...v0.15.13

1. fix(plugin-chart-echarts): disable pie chart animation (apache-superset/superset-ui#816)
2. feat: update time_compare description and choices (#818) 
3. fix(plugin-chart-echarts): improve yAxisBounds parsing (apache-superset/superset-ui#815)
4. feat(plugin-chart-echarts): add support for formula annotations (apache-superset/superset-ui#817)
5. feat(legacy-plugin-chart-sankey): allow sort by metric (apache-superset/superset-ui#831)

### TEST PLAN

Manual and CI

### ADDITIONAL INFORMATION

- [x] Has associated issue: 
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by ktmud** [comment]
@etr2460 @villebro weird enough, I'm not seeing the `mathjs` test failing after upgrading all packages.

**Comment by villebro** [comment]
@ktmud very strange. Well, let's get this merged and see what happens when I rebase the boxplot PR.

File: superset/viz.py [hunk 1]
```diff
@@ -1854,6 +1854,8 @@ def query_obj(self) -> QueryObjectDict:
                 _("Pick exactly 2 columns as [Source / Target]")
             )
         qry["metrics"] = [self.form_data["metric"]]
+        if self.form_data.get("sort_by_metric", False):
```
Comment (etr2460) [inline superset/viz.py:1857]: you don't need the False here, it'll default to None which is falsey

**Review by etr2460 — APPROVED** [review]
one comment, otherwise lgtm

File: superset/viz.py (hunk 1 above)
Comment (ktmud) [reply superset/viz.py:1857]: I thought supplying a default helps indicating this parameter is a boolean.

Since CI is green, I think I'll leave it.

**Pull request merged**